testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveStorageFormats.java

```diff
@@ -1156,4 +1258,20 @@ public String getReadType(HiveTimestampPrecision precision)
     {
         return io.trino.jdbc.Row.builder();
     }
+
+    private static class StorageFormatsHiveCompatibilityColumnData
+    {
+        private final String columnName;
```

Comment:
columnName could be generated, perhaps like

```
format("col_%s_%s", i, trinoColumnType.replaceAll("[^a-z0-9]", ""))
```

Reply:
we use the `columnName` in the `SELECT` statement done via hive jdbc in order to leave out the types (`map`, `array`, `struct` ) which default to `String`. I think there exists value in being able to specify the column name explicitly. 

In any case, this is not a repetitive operation (there's only one row being inserted) at the moment.
I'd rather stick with the current implementation.